google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryErrorMessages.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery;
+
+public class BigQueryErrorMessages {
+  public static final String RATE_LIMIT_EXCEEDED_MSG =
+      "Exceeded rate limits:"; // Error Message for RateLimitExceeded Error
```

Comment:
In addition, is there an error msg for "backendError" that we can add here which is similar to the [python client](https://github.com/googleapis/python-bigquery/blob/6502a602337ae562652a20b20270949f2c9d5073/google/cloud/bigquery/retry.py#L36)?

Reply:
Since it is hard to know what error message `backendError` manifests to, we will hold this back for now.